src/conditions/vrl.rs

```diff
@@ -12,6 +14,9 @@ use crate::{
 #[derive(Deserialize, Serialize, Debug, Default, Clone, PartialEq)]
 pub struct VrlConfig {
     pub(crate) source: String,
+
+    #[serde(default)]
```

Comment:
Looking at the soaks: does this currently mean `::Vm`? 🤔

I thought we hadn't flipped over the default yet, and should align with that before merging.

Reply:
See the latest soak tests which have no difference.

Reply:
Yeah, it's back to Ast now.

Reply:
Oh. I'm blind 😆